tools/make-self-signed-cert.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -x
+
 openssl genrsa -out privatekey.pem 2048
-openssl req -new -sha256 -key privatekey.pem -out csr.pem
+openssl req -new -sha256 -key privatekey.pem -out csr.pem -subj '/CN=www.toizom.com/O=MozIoT Gateway/C=US'
```

Comment:
This is just to generate the certificate without requiring user interaction. I just picked a name. It seems to work fine. I tested it locally, and it seems to still just treat it as self-signed. I think that any data would be fine.